src/app/pages/session-lobby/session-lobby.component.ts

```diff
@@ -12,28 +15,34 @@ import { Router } from '@angular/router';
 })
 export class SessionLobbyComponent implements OnInit {
 
-
-  sessionID = 'demo-session'; // Temporary until a way to get the session is implemented
-  roundData = {flowerSpeciesIds: ['asclepias_syriaca', 'coreopsis_palmata'],
-                        status: 'fine',
-                        running: false,
-                        currentTime: this.teacherRoundService.startTime.time, };
-
   constructor(
     public teacherSessionService: TeacherSessionService,
     public teacherRoundService: TeacherRoundService,
-    public router: Router
+    public router: Router,
+    public matDialog: MatDialog
   ) {  }
 
   ngOnInit(): void {
   }
 
-  public startRound() {
-    this.teacherRoundService.startNewRound(this.roundData);
+  public startRound(template: RoundTemplate) {
+    this.teacherRoundService.startNewRound(template);
   }
 
   public quitSession() {
     this.teacherSessionService.leaveSession();
     this.router.navigate(['host']);
   }
+
+  openRoundDialog(): void {
+    const dialogRef: MatDialogRef<RoundChooserDialogComponent, RoundTemplate> =
+      this.matDialog.open(RoundChooserDialogComponent);
+
+    dialogRef.afterClosed().subscribe(template => {
+      console.log('The dialog was closed with template ' + template?.name);
```

Comment:
Is this printout meant to be left there or was it just for development purposes?

Reply:
Ah, I believe that was just for development purposes. The dialog is confirmed to act as intended, so we should probably delete it.